src/app/_components/Footer/Footer.js

```diff
@@ -1,5 +1,101 @@
-import React from 'react';
+"use client";
+
+import React from "react";
+import Image from "next/image";
+import Link from "next/link";
+import OutlinedBtn from "../OutlinedBtn";
+import * as FooterImg from "../../../../public/assets/icons/white-logo.svg";
+import InstagramComponent from "../Icons/Instagram";
+import FacebookComponent from "../Icons/Facebook";
+import XComponent from "../Icons/XSocial";
 
 export default function Footer() {
-  return <footer>Footer</footer>;
+  // Basic styles
+  const flexBetween = "flex flex-row items-center justify-between";
+  const flexCenter = "flex flex-row items-center justify-center";
+  const basicLink = "no-underline list-none cursor-pointer";
+  const iconSizes = "w-[24px] h-[24px]";
+  const iconColors = "text-other-white hover:text-primary-400";
+  const transition = "transition duration-200 ease-in-out";
+  const footerBtnHoverFocus =
+    "hover:bg-gray-100 hover:text-primary-500 focus:bg-gray-100 focus:text-primary-600 focus:border-gray-100 active:bg-other-white active:text-primary-600 active:border-other-white";
+
+  return (
+    <footer className="text-white flex w-full flex-col bg-primary-800 px-20 py-12">
+      <div className={`${flexBetween}`}>
```

Comment:
@Yivgha let's refactor all the class interpolation into the usage of the `cn` utility.

Take inspiration from here -> https://github.com/shadcn-ui/ui/blob/0fae3fd93ae749aca708bdfbbbeddc5d576bfb2e/templates/next-template/lib/utils.ts#L2

TwMerge is an important piece of the workflow with Tailwind. It fixes all the problems related to Tailwind style conflicts.

Reply:
Good advice! Created cn util fn.